test/cypress/support/commands.js

```diff
@@ -605,27 +608,44 @@ Cypress.Commands.add('randomPaymentMethodInIncomeDetails', () => {
   })
 })
 
-Cypress.Commands.add('giAddNewChatroom', (
-  name, description = '', isPrivate = false
-) => {
+Cypress.Commands.add('giAddNewChatroom', ({
+  name, description = '', isPrivate = false, bypassUI = false
+}) => {
```

Comment:
I noticed `giAddNewChatroom` command is used many times in succession in `group-chat.spec.js` but it didn't have `bypassUI: true` option implemented yet. So I thought I would implement&use it here to make that test-suite take shorter time.